pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -375,11 +371,7 @@ func (ssc *defaultStatefulSetControl) updateStatefulSet(
 	}
 
 	if unhealthy > 0 {
-		klog.V(4).Infof("StatefulSet %s/%s has %d unhealthy Pods starting with %s",
-			set.Namespace,
-			set.Name,
-			unhealthy,
-			firstUnhealthyPod.Name)
+		klog.V(4).InfoS("StatefulSet has unhealthy Pods", "statefulSet", klog.KObj(set), "unhealthyPodNumber", unhealthy, "firstUnhealthyPod", klog.KObj(firstUnhealthyPod))
```

Comment:
Please compare this key `unhealthyPodNumber` with ones proposed on line 115-121 (`ready`, `current`, `updated`...). Values for both keys represent number of pods in particular state, however they use different naming style.

One of the important parts of the structured logging guidelines is consistency, with a goal of key names being predictable and understandable without full context.

Could you pick one naming convention and update all cases? I would prefer more verbose option so `unhealthyPodNumber`. This means that we should update `ready` -> `readyPodNumber` etc

Reply:
ok